.circleci/config.yml

```diff
@@ -48,7 +48,7 @@ jobs:
       - checkout
       - run:
           command: |
-            pip install --user --progress-bar off jinja2
+            pip install --user --progress-bar off jinja2 pyyaml
```

Comment:
This seems to be the only non-cosmetic change to this file, is this the case? It would have been easier to read if all the other changes were done in a separate PR.

Reply:
True.  I didn't realize I needed this addition until late in the process.